xarray_beam/_src/zarr_test.py

```diff
@@ -253,6 +253,68 @@ def test_chunks_to_zarr(self):
     ):
       inputs2 | xbeam.ChunksToZarr(temp_dir, template)
 
+  def test_chunks_to_zarr_append(self):
+    zarr_chunks = {'t': 1, 'x': 5}
+
+    # ds is the full dataset we want to write to zarr.
+
+    # Warning: Do not chunk this ds. If you do, it will secretly
+    # write itself to disk, then simply modifying the template (without
+    # even calling ChunksToZarr) is enough to make all tests pass!
```

Comment:
This looks like a potentially serious bug -- maybe worth filing an issue with a minimal reproducer?

Reply:
I stepped through the code and watched the file mod times. They are all as they should be. So I think caching is to blame. I updated the comment.